labels.py

```diff
@@ -0,0 +1,26 @@
+from cProfile import label
+from lib2to3.pytree import LeafPattern
+from Labelling.config.constants import TEST_IMG_DIR_PATH
+
+import shutil
+import os
+
+
+LABEL_DIR = TEST_IMG_DIR_PATH
+NEW_PATH = os.path.abspath('Labels')
+
+print(NEW_PATH)
+
+for path, dirs, files in os.walk(LABEL_DIR):
+    for file in files:
+        if file.endswith('json'):
+            OLD_LOC = os.path.join(path, file)
```

Comment:
Just a general note about `SCREAMING_SNAKE_CASE` (I got the name from [here](https://en.wikipedia.org/wiki/Naming_convention_(programming)#:~:text=snake_case%2C%20pothole_case-,TWO_WORDS,SCREAMING_SNAKE_CASE%2C%20MACRO_CASE%2C%20CONSTANT_CASE,-two_Words), it's generally reserved for global constants in Python. I may have abused them in other places in this project (sorry), but I'm trying to clean it up haha! 😂 )